src/entity/Alerts.ts

```diff
@@ -18,6 +18,9 @@ export class Alerts {
   @Column({ type: 'bool', default: true })
   companionHelper: boolean;
 
+  @Column({ type: 'bool', default: true })
+  showSquadTour: boolean;
```

Comment:
Actually looking at the other we don't prefix `show` so maybe `squadTour` is enough?

Reply:
Agree with this convention. Let's fix to `squadTour`

Reply:
Good point. Let me refactor it :ship